highcharts-api/highcharts/7-connecting-column-edges/main.js

```diff
@@ -0,0 +1,54 @@
+const generateRandomData = (length) => Array.from({ length }, () => Math.random() * 14);
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'column',
+
+        events: {
+            render() {
+                const chart = this;
+                console.log(this);
+
+                let zIndex = 3;
+                
+                for(data of chart.series) {
```

Comment:
I would suggest using the `forEach()` method here.

```suggestion
                chart.series.forEach(function(series) {
```